src/Pages/3DPrintingConsole/requestForm.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import './3D-console.css'
+import {
+  Button,
+  ButtonGroup,
+  Card,
+  CardTitle,
+  CardBody,
+  Collapse,
+  FormGroup,
+  Row,
+  Col
+} from 'reactstrap'
+
+function RequestForm (props) {
+  const jsonObject = props.item
+  const key = props.item._id
+
+  const Console3DLabel = [
+    ['3', 'E-mail/Contact:'],
+    ['3', 'Requested Date:'],
+    ['2', 'Progress']
+  ]
+  const Console3DValues = [
+    jsonObject.projectContact,
+    jsonObject.date,
+    jsonObject.progress
+  ]
+  const Console3DButtons = [
+    {
+      color: 'primary',
+      value: 'Reject',
+      handleUpdates: e => {
+        props.onHandleUpdateProgress(jsonObject, e)
+      }
+    },
+    {
+      color: 'info',
+      value: 'In Progress',
+      handleUpdates: e => {
+        props.onHandleUpdateProgress(jsonObject, e)
+      }
+    },
+    {
+      color: 'secondary',
+      value: 'Completed',
+      handleUpdates: e => {
+        props.onHandleUpdateProgress(jsonObject, e)
+      }
+    },
+    {
+      color: 'primary',
+      value: 'Delete',
+      handleUpdates: e => {
+        props.onHandleDeleteData(jsonObject)
+      }
+    }
+  ]
+
+  return (
+    <FormGroup key={key}>
```

Comment:
we aren't mapping so you dont need a key here. don't send the key as a prop and remove it here

Reply:
It's being returned and mapped out in 3DConsole.js

Reply:
Right, however you only need a `key` _within_ the `map` function, which you already have [here](https://github.com/SCE-Development/Core-v4/blob/7ee14c2aad6f70c857c45a602495aa969ca0f30b/src/Pages/3DPrintingConsole/3DConsole.js#L100). There is no need to have it in this file.